src/test_encode_decode_dav1d.rs

```diff
@@ -24,8 +24,8 @@ pub(crate) struct Dav1dDecoder<T: Pixel> {
 impl<T: Pixel> TestDecoder<T> for Dav1dDecoder<T> {
   fn setup_decoder(_w: usize, _h: usize) -> Self {
     unsafe {
-      let mut settings = mem::uninitialized();
-      let mut dec: Dav1dDecoder<T> = mem::uninitialized();
+      let mut settings = MaybeUninit::init().assume_init();
```

Comment:
`MaybeUninit` is missing from includes. The same change from `test_encode_decode_aom.rs` is needed. Since this is not compiled by default it won't generate an error unless the feature is enabled.